tests/test_slippage.py

```diff
@@ -24,75 +24,138 @@
 
 from nose_parameterized import parameterized
 
+import numpy as np
 import pandas as pd
+from testfixtures import TempDirectory
 
 from zipline.finance.slippage import VolumeShareSlippage
+from zipline.finance.trading import TradingEnvironment, SimulationParameters
 
 from zipline.protocol import Event, DATASOURCE_TYPE
 from zipline.finance.blotter import Order
 
+import zipline.utils.factory as factory
+from zipline.data.minute_writer import MinuteBarWriterFromDataFrames
+from zipline.data.data_portal import DataPortal
+
 
 class SlippageTestCase(TestCase):
 
-    def test_volume_share_slippage(self):
-        event = Event(
-            {'volume': 200,
-             'type': 4,
-             'price': 3.0,
-             'datetime': datetime.datetime(
-                 2006, 1, 5, 14, 31, tzinfo=pytz.utc),
-             'high': 3.15,
-             'low': 2.85,
-             'sid': 133,
-             'source_id': 'test_source',
-             'close': 3.0,
-             'dt':
-             datetime.datetime(2006, 1, 5, 14, 31, tzinfo=pytz.utc),
-             'open': 3.0}
+    @classmethod
+    def setUpClass(cls):
+        cls.tempdir = TempDirectory()
+        cls.env = TradingEnvironment()
+
+        cls.sim_params = SimulationParameters(
+            period_start=pd.Timestamp("2006-01-05 14:31", tz="utc"),
+            period_end=pd.Timestamp("2006-01-05 14:36", tz="utc"),
```

Comment:
A slight hesitation on these sim params is that they are 'non-standard', as in that we never run simulations of that length.

On the other hand, having only 6 data points is easy to reason about.